### PR TITLE
[dev] Bug AB#33063: [Contact Information] Disable international region code for phone numbers

### PR DESCRIPTION
**Pull request by raoufr**
…opdown.jsx

### CODE REVIEW

File: src/inputs/phoneInputCountryDropdown.jsx [hunk 1]
```diff
@@ -63,13 +63,16 @@ class PhoneInputCountryDropdown extends React.PureComponent {
...
-        });
+        if (selectedOption) {
+            const { onChange } = this.props;
+
+            this.setState({
+                selectedOption,
+            }, () => {
+                onChange(selectedOption.value);
+            });
+        }
```
Comment (morethanfire) [inline src/inputs/phoneInputCountryDropdown.jsx:75]: So the idea here is to be defensive?

**Comment by morethanfire** [comment]
@raoufr, can you please fix the conflict?

**Comment by groberts314** [comment]
PR Review Checkpoint Meeting Thurs 08/12/2020.
Raouf to resolve conflict and then :shipit:

File: src/inputs/phoneInputCountryDropdown.jsx (hunk 1 above)
Comment (raoufr) [reply src/inputs/phoneInputCountryDropdown.jsx]: yes @morethanfire

**Comment by raoufr** [comment]
> @raoufr, can you please fix the conflict?
@morethanfire  done

**Pull request merged**